<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>111</width>
      <height>321</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="tbtnLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/Line-Tool-512.png</normaloff>:/icons/icons/Line-Tool-512.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnStamp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/stamp-512.png</normaloff>:/icons/icons/stamp-512.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnBrush">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/brush.png</normaloff>:/icons/icons/brush.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnFill">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/fill-color.png</normaloff>:/icons/icons/fill-color.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/Editing-Text-icon.png</normaloff>:/icons/icons/Editing-Text-icon.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnSpray">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/spray.png</normaloff>:/icons/icons/spray.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnForegroundColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnBackgroundColor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtnSwapColors">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/swap.png</normaloff>:/icons/icons/swap.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Datei">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="action_Speichern_unter"/>
    <addaction name="separator"/>
    <addaction name="action_Beenden"/>
   </widget>
   <widget class="QMenu" name="menu_Extras">
    <property name="title">
     <string>&amp;Extras</string>
    </property>
    <addaction name="action_Optionen"/>
   </widget>
   <widget class="QMenu" name="menu_Hilfe">
    <property name="title">
     <string>&amp;Hilfe</string>
    </property>
    <addaction name="action_ber_PaintIt"/>
   </widget>
   <addaction name="menu_Datei"/>
   <addaction name="menu_Extras"/>
   <addaction name="menu_Hilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_ffnen">
   <property name="text">
    <string>&amp;Öffnen</string>
   </property>
  </action>
  <action name="action_Speichern_unter">
   <property name="text">
    <string>&amp;Speichern unter</string>
   </property>
  </action>
  <action name="action_Beenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
  <action name="action_Optionen">
   <property name="text">
    <string>&amp;Optionen</string>
   </property>
  </action>
  <action name="action_ber_PaintIt">
   <property name="text">
    <string>&amp;Über PaintIt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
